experimenter/tests/integration/nimbus/utils/helpers.py

```diff
@@ -142,7 +142,16 @@ def load_config_data():
 
 
 def load_targeting_configs(app=BaseExperimentApplications.FIREFOX_DESKTOP.value):
-    config_data = load_config_data()
+    config_data = []
+
+    for retry in range(LOAD_DATA_RETRIES):
+        try:
+            config_data = load_config_data()
+        except requests.exceptions.JSONDecodeError:
+            if retry + 1 >= LOAD_DATA_RETRIES:
+                raise
+            time.sleep(LOAD_DATA_RETRY_DELAY)
```

Comment:
But we're already retrying in the `load_graphql_data` function, which this calls, shouldn't we just increase the number of retries?

Reply:
Oh yeah, you're right. For some reason I thought this was failing instantly but I see now it's about 10 seconds which is the length of the retries we have. Ill just increase that.